package.json

```diff
@@ -33,11 +33,13 @@
     "ember-disable-prototype-extensions": "^1.1.0",
     "ember-disable-proxy-controllers": "^1.0.1",
     "ember-export-application-global": "^1.0.4",
+    "ember-get-config": "0.0.2",
```

Comment:
I'd also change the version to be `>=0.0.2 <2.0.0` so consumers get newer versions.